client/tests/webdriver/baseSpecs/createNewPerson.spec.js

```diff
@@ -208,17 +202,11 @@ describe("Create new Person form page", () => {
       // This makes sure the help-block is displayed after form submit
```

Comment:
This comment is now no longer accurate (nothing will be shown since empty values are allowed).